src/tools/debugger/stepping/Stepping.java

```diff
@@ -0,0 +1,42 @@
+package tools.debugger.stepping;
+
+import tools.SourceCoordinate.FullSourceCoordinate;
+/**
+ *
+ * Manage the actor message stepping.
+ *
+ */
+public class Stepping {
+
+  public enum SteppingType{
+    STEP_INTO,
+    STEP_OVER,
+    STEP_RETURN
+  }
+
+  private StepActorOperation stepActor;
+
+  public Stepping(final StepActorOperation stepActor) {
+    this.stepActor = stepActor;
+  }
+  /**
+   * Return the stepping operation that has been made for a specific source section,
```

Comment:
hmmm, this all still looks strange. I am not really sure what you want to achieve with that method in the first place, you use that in the end to set the type of stepping strategy on the message? I am confused. sorry, won't be able to look into this in detail today.